Src/BootCamp.Chapter/Inventory.cs

```diff
@@ -1,35 +1,58 @@
-﻿namespace BootCamp.Chapter
+﻿using System.Collections.Generic;
+
+namespace BootCamp.Chapter
 {
     public class Inventory
     {
-        private Item[] _items;
-        public Item[] GetItems()
+        private List<Item> _items;
```

Comment:
can remove the private backing field and just use an auto-property

Reply:
As recommended, I have removed the private backing field. and am using the auto-implemented properities.